wemake_python_styleguide/violations/naming.py

```diff
@@ -479,3 +480,46 @@ class ConsecutiveUnderscoresInNameViolation(MaybeASTViolation):
 
     error_template = 'Found consecutive underscores name: {0}'
     code = 116
+
+@final
+class TooLongNameViolation(ASTViolation):
+    """
+    Forbids to have too long variable or module names.
+
+    Reasoning:
+        Too long names are unreadable,
+        it is better to use a shorter alternative.
+        Long names also indicate that this variable is too complex,
+        it may require some documentation.
+
+    Solution:
+        Think of a shorter, more concise name.
+
+    This rule checks: modules, variables, attributes,
+    functions, methods, and classes.
+
+    Example::
+
+        # Correct:
+        total_price = 100
+        courses_enrolled = 5
+
+        # Wrong:
+        subtotal_plus_taxes_and_gratuity_on_top_at_15_percent = 100
+        number_of_full_time_courses_plus_number_of_part_time_courses = 5
+
+    Configuration:
+        This rule is configurable with ``--max-name-length``.
+        Default:
+        :str:`wemake_python_styleguide.options.defaults.MAX_NAME_LENGTH`
+
+    .. versionadded:: 0.4.0
+
+    Note:
+        Returns Z117 as error code
+
+    """
+
+    #: Error message shown to the user.
```

Comment:
We do not use this comment anymore, see this commit https://github.com/wemake-services/wemake-python-styleguide/commit/e428aab6e79409b9a10dd2134cd1b8685bdc2213